### PR TITLE
We are bridging the first ERC20 token ever (MistCoin) to Base!

### PR DESCRIPTION
**Pull request by esvictorgc**
We are bridging the first ERC20 token deployed in Ethereum, released by Fabian Vogelsteller and Alex Van de Sande, to Base!

A clear and concise description of the features you're adding in this pull request.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.
@cfluke-cb 

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Comment by esvictorgc** [comment]
hi, can you help us guys? When this request will be reviewed?

File: data/WMC/data.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Wrapped MistCoin",
+  "symbol": "WMC",
+  "decimals": "2",
```
Comment (njokuScript) [inline data/WMC/data.json:4]: can you change this to a number. from "2" => 2 @esvictorgc

Comment (esvictorgc) [reply data/WMC/data.json]: Done! thank you

Comment (esvictorgc) [reply data/WMC/data.json]: @cfluke-cb can you check if we are OK not to go live? thanks base team!

**Review by wbnns — APPROVED** [review]
ACK

**Pull request merged**